### DISCUSSION TITLE
Combining Mobility Models - Showcase

### DISCUSSION THREAD

**Discussion by avarga** [inet-framework/inet#608] [Showcases]
Discuss the "Combining Mobility Models" showcase here. Ideas and recommendations are welcome.

**Comment by NetColby** [comment]
Hello, is it possible to combine multiple mobility models based on the nodal status? E.g., each node follows two mobility models, one static model and one linear mobility. When a node's energy level is above a threshold, it moves linearly. Otherwise, it stays still.  Thanks!

**Comment by levy** [comment]
It's a reasonable expectation to have a parameter in mobility models which says what should the node do if the energy storage is depleted. For example, a drone should just stop, but a satellite should continue its motion. It's not implemented, but I'll put this on the TODO list.

**Comment by ajp1871** [comment]
Hi, I reported an issue with the Mars Rover example of the combining mobility model in the general issue section earlier today. Just found the link to this discussion while reading through the documentation. 

Here's the link to my issue: https://github.com/inet-framework/inet/issues/489#issue-530722985. Perhaps you know of this issue? Regards!

**Comment by adalberto1986** [comment]
I want to implement mobile LoRa nodes in FloRA combined with INET mobility, and I don't know how to proceed to define
